v8go.cc

```diff
@@ -1207,6 +1207,19 @@ RtnValue FunctionNewInstance(ValuePtr ptr, int argc, ValuePtr args[]) {
   return rtn;
 }
 
+RtnValue FunctionSourceMapUrl(ValuePtr ptr) {
+  LOCAL_VALUE(ptr)
+  RtnValue rtn = {nullptr, nullptr};
+  Local<Function> fn = Local<Function>::Cast(value);
+  Local<Value> result = fn->GetScriptOrigin().SourceMapUrl();
+  m_value* rtnval = new m_value;
+  rtnval->iso = iso;
+  rtnval->ctx = ctx;
+  rtnval->ptr = Persistent<Value, CopyablePersistentTraits<Value>>(iso, result);
+  rtn.value = tracked_value(ctx, rtnval);
+  return rtn;
+}
```

Comment:
The return value is still implying that this could return an error, but there isn't a code path that would return an error.

```suggestion
ValuePtr FunctionSourceMapUrl(ValuePtr ptr) {
  LOCAL_VALUE(ptr)
  RtnValue rtn = {nullptr, nullptr};
  Local<Function> fn = Local<Function>::Cast(value);
  Local<Value> result = fn->GetScriptOrigin().SourceMapUrl();
  m_value* rtnval = new m_value;
  rtnval->iso = iso;
  rtnval->ctx = ctx;
  rtnval->ptr = Persistent<Value, CopyablePersistentTraits<Value>>(iso, result);
  return tracked_value(ctx, rtnval);
}
```